presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -216,6 +216,8 @@ Property Name                                      Description
 
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
+``hive.insert-existing-partitions-behavior``       Default value for insert existing partitions behavior.       ``APPEND``
```

Comment:
Please mention other possible values. Also you could shortly explain what each of these options mean.

Reply:
done